### DISCUSSION TITLE
Placeholder counts question

### DISCUSSION THREAD

**Discussion by JonatanLaMerce** [TownyAdvanced/Towny#5252] [Q&A] [answered]
Hi all, is there any way or placeholder to count the players that are inside a nation or city, without the need for the player to be inside the nation or city?

Something similar to %townyadvanced_town_residents_amount% and %townyadvanced_nation_residents_amount% but without the need for the player to be inside the nation or city. 

thanks!

**Comment by LlmDl** [comment, marked as answer]
The placeholders you listed above apply to the player they are used upon. Discussions are not where we take [suggestions](https://github.com/TownyAdvanced/Towny/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=Suggestion%3A+).
